### ISSUE TITLE
`module` template throws error and crashes CLI

### ISSUE THREAD

**Issue by exaby73** [serverpod/serverpod#225]
###### Logs

```
Unhandled exception:
ProcessException: No such file or directory
  Command: ./setup-tables
#0      _ProcessImpl._start (dart:io-patch/process_patch.dart:401:33)
#1      Process.start (dart:io-patch/process_patch.dart:38:20)
#2      CommandLineTools.createTables (file:///Users/nabeelparkar/.pub-cache/hosted/pub.dartlang.org/serverpod_cli-0.9.8/bin/util/command_line_tools.dart:58:33)
<asynchronous suspension>
#3      performCreate (file:///Users/nabeelparkar/.pub-cache/hosted/pub.dartlang.org/serverpod_cli-0.9.8/bin/create/create.dart:340:5)
<asynchronous suspension>
#4      main (file:///Users/nabeelparkar/.pub-cache/hosted/pub.dartlang.org/serverpod_cli-0.9.8/bin/serverpod_cli.dart:220:9)
<asynchronous suspension>
```

**Comment by vlidholt** [comment]
Thanks! Will fix.

**Comment by yahu1031** [comment]
Yes, Windows doesn't have support completely and therefore there is no `setup-tables` file in windows yet. I'm on it. I can make a PR once it is done.

**Comment by exaby73** [comment]
@yahu1031 Thanks for the reply. Though I should note, I am running this on an M1 Macbook Air.

**Comment by yahu1031** [comment]
Okie i can check once and also please try posting the full logs.

**Comment by yahu1031** [comment]
Looks like `setup-tables` file doesn't copy/created there.

**Comment by yahu1031** [comment]
Fix added in #234

**Comment by vlidholt** [comment]
Fixed
